src/main/java/Parser.java

```diff
@@ -64,95 +76,203 @@ public boolean createIndex(String file, String indexDirectoryLocation) throws IO
         config.setOpenMode(IndexWriterConfig.OpenMode.CREATE);
         IndexWriter indexWriter = new IndexWriter(directory, config);
 
+        String[] locations = {fbisLocation, latimesLocation, fr94Location};
+        boolean successful = indexCollections(locations, indexWriter) && indexFt(ftLocation, indexWriter);
```

Comment:
why do we parse the financial times corpus separately here?

Reply:
FT was already done by @jackgilbride999 before I started. Now that we have made a common function, @jackgilbride999 Can you make it use the common function? Or we could sit together for this.

Reply:
@PathakPratik for this, will I just add the ftLocation to the locations array?

Reply:
Even I think that should work. But just added you in the loop so no other functionality breaks.

Reply:
The number of fields we are indexing is different. That is one thing that we might have to discuss

Reply:
I think that is okay for the moment. I just added the headline in as it felt sensible at the time, but it is probably best to keep it consistent with the other three corpus for now.